### ISSUE TITLE
Cannot run due to missing VERSION file

### ISSUE THREAD

**Issue by jat255** [bartfeenstra/betty#937]
I created a new virtual environment in which to run `betty`:

```
$ python -m venv .venv
```

Activated it:

```
$ source .venv/bin/activate
```

Installed betty:

```
(.venv) $ pip install betty
```

But when trying to run the software, I get the following error:

```
(.venv) $ betty
Traceback (most recent call last):
  File "/home/user/.venv/bin/betty", line 5, in <module>
    from betty.cli import main
  File "/home/user/.venv/lib/python3.10/site-packages/betty/cli.py", line 148, in <module>
    @click.version_option(about.version(), prog_name='Betty')
  File "/home/user/.venv/lib/python3.10/site-packages/betty/about.py", line 5, in version
    with open(path.join(path.dirname(path.dirname(__file__)), 'VERSION')) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/.venv/lib/python3.10/site-packages/VERSION'
```

Any ideas what's gone wrong?

**Comment by jat255** [comment]
It looks like there was an issue similar to this in 2020: https://github.com/bartfeenstra/betty/pull/702

I worked around it by running:

```
$ echo 0.2.6 > .venv/lib/python3.10/site-packages/betty/VERSION
```

And changing `about.py` to

```python
from os import path


def version() -> str:
    with open(path.join(path.dirname(__file__), 'VERSION')) as f:
        release_version = f.read().strip()
    if release_version == '0.0.0':
        return 'development'
    return release_version
```

**Comment by jat255** [comment]
After some more futzing, I realized the 0.2.6 branch is pretty out of date, but seeing as how this is an issue installing straight from PyPI, I'll leave the issue open.

**Comment by bartfeenstra** [comment]
Hi there! Thanks for reporting this. I managed to reproduce the problem and am working on a fix.

**Comment by bartfeenstra** [comment]
The code is correct (installing Betty from source works). What appears to happen is that the `VERSION` file is not correctly bundled into the PyPI distributions. I'm still searching for the source of this problem.

**Comment by bartfeenstra** [comment]
@jat255 Thanks again for your report. Could you test the fix by following the *How to test* instructions at https://github.com/bartfeenstra/betty/pull/969, and post your feedback there?

**Comment by jat255** [comment]
Appears to work for me!

**Comment by bartfeenstra** [comment]
Thanks! I just released Betty 0.2.7 with this fix, and [several others](https://github.com/bartfeenstra/betty/milestone/9?closed=1).
